run/start-all.mjs

```diff
@@ -60,11 +60,25 @@ async function main() {
     gatewayServers[serverName] = fork('../services/main/gateway-server.mjs', ['--service-name', serverName], { cwd: '.' });
   }
 
+  // Wait for all gateway services to start.
+  const promises = Object.keys(gatewayServers).map((serverName) => {
+    return new Promise((resolve, reject) => {
+      gatewayServers[serverName].on('message', msg => {
+        if(msg !== 'started') return;
+        console.log(`${serverName} has started successfully.`)
+        resolve(msg)
+      });
+      const timer = setTimeout(() => {
+        reject(new Error(`Promise timed out after 10 sec.`));
+      }, 10 * 1000);
+    });
+  });
```

Comment:
```javascript
const primises = Object.entries(gatewayServers).map((entry) => {
    const [name, server] = entry;
    server.on(...);
    ...
})

```

Reply:
Error message should be more informative. For example, `Failed to start "${name}" server successfully in ${timeout} seconds.`.